lib/trento/clusters/value_objects/cluster_resource_parent.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Trento.Clusters.ValueObjects.ClusterResourceParent do
+  @moduledoc """
+  Represents the parent of a cluste resource
```

Comment:
```suggestion
  Represents the parent of a cluster resource
```
**typo:** `cluste` -> `cluster`